share/spack/gitlab/cloud_pipelines/stacks/hep/spack.yaml

```
@@ -0,0 +1,64 @@
+spack:
+  view: false
+
+  concretizer:
+    reuse: false
+    unify: when_possible
+
+  packages:
+    all:
+      require: '%gcc target=x86_64_v3'
+      providers:
+        blas: [openblas]
+        mpi: [mpich]
+        tbb: [intel-tbb]
+      variants: +mpi
+
+    # Mark geant4 data as external to prevent wasting bandwidth on GB-scale files
+    geant4-data:
+      externals:
+        - spec: geant4-data@11.3.0
+          prefix: /usr
+        - spec: geant4-data@11.2.2
+          prefix: /usr
+        - spec: geant4-data@11.2.0
+          prefix: /usr
+        - spec: geant4-data@11.1.0
+          prefix: /usr
+        - spec: geant4-data@11.0.0
+          prefix: /usr
+
+  specs:
+  # CPU
+  - acts +analysis +dd4hep +edm4hep +examples +fatras +geant4 +hepmc3 +podio +pythia8 +python +tgeo cxxstd=20
+  - dd4hep +ddalign +ddcad +ddcond +dddetectors +dddigi +ddeve +ddg4 +ddrec +edm4hep +hepmc3 +lcio +utilityapps +xercesc
+  - delphes +pythia8
+  - edm4hep
+  - fastjet
+  - fjcontrib
+  - gaudi +aida +examples +heppdt +xercesc
```

Without explicit `gdb +python`, we concretize on `gdb@8.1`... which fails.
```suggestion
  - gaudi +aida +examples +heppdt +xercesc ^gdb +python
```